data/colors/dark_dimmed.ts

```diff
@@ -1,86 +0,0 @@
-import {merge} from '../../src/utils'
```

Comment:
Does this deletion count as major? Or was the direct `data/colors/dark_dimmed` never used by external users?

Reply:
These source files are never referenced directly so it has no impact on consumers.